plugins/kernel_v2/config/vm_provisioner.rb

```diff
@@ -69,6 +75,7 @@ def initialize(name, type, **options)
         @type    = type
         @before  = options[:before]
         @after   = options[:after]
+        @communicator_required = options[:communicator_required] || true
```

Comment:
Should probably be `options.fetch(:communicator_required, true)` to allow the option to set it to `false`